common/src/main/java/nl/tudelft/trustchain/common/eurotoken/TransactionRepository.kt

```diff
@@ -524,63 +505,6 @@ class TransactionRepository(
         }
     }
 
-    fun verifyBalanceAvailable(
```

Comment:
Why did this need to be deleted?

Reply:
Because it Android studio showed that it was completely unused. Probably it was used by some module in the past, that does not exist after the cleanup. Should I restore it?

Reply:
We could keep it in for now. Seems useful

Reply:
Yeap! I pushed a commit that restores these 2 functions